File: CPP/BEE-1049_Animal.cpp
/*-------------------*
| Rodrigo CavanhaMan |
| URI 1049           |
| Animal             |
*--------------------*/
#include<stdio.h>
#include <iostream>
#include <string>
using namespace std;

int main() {

	string AN1, AN2, AN3;
	getline(cin, AN1);
	getline(cin, AN2);
	getline(cin, AN3);

	if (AN1=="vertebrado") {
		if (AN2=="ave") {
			if (AN3=="carnivoro") 
				cout << "aguia\n";
			else
				cout << "pomba\n";
		}
		else {
			if (AN3=="onivoro")
				cout << "homem\n";
			else
				cout << "vaca\n";
		}
	}
	else {
		if (AN2=="inseto") {
			if (AN3=="hematofago")
				cout << "pulga\n";
			else
				cout << "lagarta\n";
		}
		else {
			if (AN3=="hematofago")
				cout << "sanguessuga\n";
			else
				cout << "minhoca\n";
		}
	}
	return 0;
}

File: BEE-1049_Animal.java
/*-------------------*
| Rodrigo CavanhaMan |
|        IFTM        |
|      BEE 1049      |
*--------------------*/
import java.io.IOException;import java.util.Locale;
import java.util.Scanner;

public class Main {

	public static void main(String[] args)  throws IOException {
		Scanner sc = new Scanner(System.in);
		sc.useLocale(Locale.ENGLISH);
		Locale.setDefault(new Locale("en", "US"));
		
		String AN1,AN2,AN3;
		
		AN1 = sc.nextLine();
		AN2 = sc.nextLine();
		AN3 = sc.nextLine();
		
		if (AN1.equals("vertebrado")) {
			if (AN2.equals("ave")) {
				if (AN3.equals("carnivoro")) {
					System.out.println("aguia");
				}
				else {
					System.out.println("pomba");
				}
			}
			else {
				if (AN3.equals("onivoro")) {
					System.out.println("homem");
				}
				else {
					System.out.println("vaca");
				}
			}
		}
		else {
			if (AN2.equals("inseto")) {
				if (AN3.equals("hematofago")) {
					System.out.println("pulga");
				}
				else {
					System.out.println("lagarta");
				}
			}
			else {
				if (AN3.equals("hematofago")) {
					System.out.println("sanguessuga");
				}
				else {
					System.out.println("minhoca");
				}
			}
		}

		sc.close();
	}
}